config/default.yml

```diff
@@ -1469,6 +1469,25 @@ Lint/DeprecatedClassMethods:
   Enabled: true
   VersionAdded: '0.19'
 
+Lint/DeprecatedConstants:
+  Description: 'Checks for deprecated constants.'
+  Enabled: pending
+  VersionAdded: '<<next>>'
+  # You can configure deprecated constants.
+  # If there is an alternative method, you can set alternative value as the first element of value.
+  # Or set `null` to the value if there is no alternative value.
+  # And you can specify the deprecated version as the second element of value.
+  # Or set `null` to the value if there is unknown deprecated version.
+  #
+  # DeprecatedConstants:
+  #   'DEPRECATED_CONSTANT': ['alternative_value', 'deprecated_version']
+  #
+  DeprecatedConstants:
+    'NIL': ['nil', '2.4']
+    'TRUE': ['true', '2.4']
+    'FALSE': ['false', '2.4']
+    'Random::DEFAULT': ['Random.new', '3.0']
```

Comment:
I'm not sure whether array is good. For example, hash may be preferable. Please let me know if you have feedback 😅

Reply:
I agree that hash would be better. This would spare us from having to use explicit `null`s.

Reply:
> This would spare us from having to use explicit `null`s.

Sure! That makes sense :-) I updated it!